wagtail/admin/templates/wagtailadmin/pages/listing/_table_headers_explore.html

```diff
@@ -17,24 +17,22 @@
 {% endcomment %}
 
 <tr class="table-headers">
-    {% if show_ordering_column %}
-        <th class="ord{% if orderable and ordering == 'ord' %} ord--active{% endif %}">
-            {% if orderable %}
-                {% if ordering == "ord" %}
-                    <a href="{% url 'wagtailadmin_explore' parent_page.id %}" title="{% trans 'Disable ordering of child pages' %}">
-                        {% icon name="order" %}{% trans 'Sort' %}
-                    </a>
-                {% else %}
-                    <a href="{% url 'wagtailadmin_explore' parent_page.id %}?ordering=ord" title="{% trans 'Enable ordering of child pages' %}">
-                        {% icon name="order" %}{% trans 'Sort' %}
-                    </a>
-                {% endif %}
-            {% endif %}
-        </th>
-    {% endif %}
-    <th class="title">
-        {% trans 'Title' as title_label %}
-        {% page_table_header_label label=title_label sortable=sortable sort_field='title' parent_page_title=parent_page.title %}
+    <th class="bulk-actions-filter-checkbox">
+        <div>
+            <input type="checkbox" aria-label="Bulk action checkbox" />
+            {% bulk_action_filters %}
+            <div class="bulk-actions-choices u-hidden">
+                <span>All 8 on this page selected. Select all 43 in listing |</span>
```

Comment:
Should we be hardcoding the 8 and 43 here? These seem quite specific. Additionally, text needs to be translatable.